manual/python/index.md

```diff
@@ -60,9 +60,87 @@ caption="Example of graphics generated with PyROOT."
+```
+
+If PyROOT cannot find `.rootlogon.py` in the user's home directory, it will look for the equivalent in C++ (`.rootlogon.C`), first in [ROOT's etc directory](https://root.cern/doc/master/classTROOT.html#ab8e51627a12d886d6c8177b46481352a), then in the user's home directory and finally in the current working directory. Note that it is also possible to use both the Python and the C++ rootlogons, since the latter can be loaded from the former, for instance with `ROOT.gROOT.LoadMacro('.rootlogon.C')`.
+
+If the user would like to completely disable the rootlogon functionality, they can do so by setting `PyConfig.DisableRootLogon` to `True`. 
+
+- `IgnoreCommandLineOptions` (default `True`): if a PyROOT script is executed with some command line arguments, they will be ignored by default by ROOT, so the user is free to process them as they wish. However, by setting `PyConfig.IgnoreCommandLineOptions` to `False`, those arguments will be forwarded to ROOT for parsing, e.g. to enable the batch mode from the command line. A complete list of the arguments accepted by ROOT can be found [here]({{ '/manual/first_steps_with_root/#starting-root-with-command-line-options' | relative_url }}).
+
+- `ShutDown` (default `True`): when PyROOT is terminating, during its cleanup phase, the ROOT C++ interpreter is shut down. If PyROOT is executed as part of a longer-running application that needs the C++ interpreter, `PyConfig.Shutdown` can be set to `False` to prevent that shutdown.
+
+- `StartGUIThread` (default `True`): when not executing in [interactive mode](https://docs.python.org/3/tutorial/interpreter.html#interactive-mode), PyROOT starts a thread that periodically polls for ROOT events (e.g. GUI events) to process them. If a given PyROOT application does not need this event processing, it can prevent the creation of the thread by setting `PyConfig.StartGUIThread` to `False`.
+
+### Enabling batch mode
+
+When running in batch mode, PyROOT will not display any graphics. There are two ways in which a user can activate the batch mode:
+
+1. Passing `-b` as a command line argument, e.g. `python -b my_pyroot_script.py`. For this to work, `PyConfig.IgnoreCommandLineOptions` must be set to `False` inside the PyROOT script, as discussed [here]({{ '/manual/python/#configuration-options' | relative_url }}).
+
+2. Calling `gROOT.SetBatch` in the PyROOT script, right after importing `ROOT`:
+
+```python
+import ROOT
+ROOT.gROOT.SetBatch(True)
+```
+
+### Low-level manipulation of objects
+
+When instantiating a C++ class from Python via PyROOT, both a C++ object and its Python proxy object are created. Such Python object forwards any access to its internal C++ object, thus acting as a proxy. PyROOT offers some functions to inspect or manipulate Python proxies and their C++ counterparts, based on functionality provided by [cppyy](https://cppyy.readthedocs.io/en/latest/lowlevel.html). All these functions can be accessed as `ROOT.NameOfFunction` and they are listed next:
```

Comment:
... can be accessed with the pattern `ROOT.NameOfFunction` ...

Just to prevent that people think `ROOT.NameOfFunction` is an actual member of ROOT.

Reply:
ok!